<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FeedWindow</class>
 <widget class="QDialog" name="FeedWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Лента</string>
  </property>
  <property name="windowIcon">
   <iconset resource="IMG.qrc">
    <normaloff>:/IMG/IMG/logoSN.png</normaloff>:/IMG/IMG/logoSN.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <widget class="QPushButton" name="openMenuButton">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>0</y>
     <width>1001</width>
     <height>71</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="IMG.qrc">
     <normaloff>:/IMG/IMG/MenuTablePin1000x70SN.png</normaloff>:/IMG/IMG/MenuTablePin1000x70SN.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>1000</width>
     <height>70</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="notificationButton">
   <property name="geometry">
    <rect>
     <x>699</x>
     <y>95</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton 
{
	background-color: white;
    border-radius: 30px;
}

QPushButton:hover
{
    background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #6a3093, 
        stop:1 #a044ff 
    );
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="IMG.qrc">
     <normaloff>:/IMG/IMG/NotificationPin50x50SN.png</normaloff>:/IMG/IMG/NotificationPin50x50SN.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="updateFeedButton">
   <property name="geometry">
    <rect>
     <x>799</x>
     <y>95</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton 
{
	background-color: white;
    border-radius: 30px;
}

QPushButton:hover
{
    background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #6a3093, 
        stop:1 #a044ff 
    );
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="IMG.qrc">
     <normaloff>:/IMG/IMG/UpdatePin50x50SN.png</normaloff>:/IMG/IMG/UpdatePin50x50SN.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
  </widget>
  <widget class="QFrame" name="menuTableFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>175</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(240, 241, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QPushButton" name="profilePinButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton 
{
	background-color: rgb(240, 241, 255);
    border-radius: 40px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="IMG.qrc">
      <normaloff>:/IMG/IMG/DefultProfile75x75SN.png</normaloff>:/IMG/IMG/DefultProfile75x75SN.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="myPostPinButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>45</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton 
{
	background-color: rgb(240, 241, 255);
    border-radius: 45px;
}

QPushButton:hover
{
    background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #6a3093, 
        stop:1 #a044ff 
    );
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="IMG.qrc">
      <normaloff>:/IMG/IMG/PostPin80x80SN.png</normaloff>:/IMG/IMG/PostPin80x80SN.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="usersPinButton">
    <property name="geometry">
     <rect>
      <x>449</x>
      <y>44</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton 
{
	background-color: rgb(240, 241, 255);
    border-radius: 45px;
}

QPushButton:hover
{
    background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #6a3093, 
        stop:1 #a044ff 
    );
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="IMG.qrc">
      <normaloff>:/IMG/IMG/UsersPin80x80SN.png</normaloff>:/IMG/IMG/UsersPin80x80SN.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="settingsPinButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>44</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton 
{
	background-color: rgb(240, 241, 255);
    border-radius: 45px;
}

QPushButton:hover
{
    background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #6a3093, 
        stop:1 #a044ff 
    );
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="IMG.qrc">
      <normaloff>:/IMG/IMG/SettingsPin80x80SN.png</normaloff>:/IMG/IMG/SettingsPin80x80SN.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="exitPinButton">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>44</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton 
{
	background-color: rgb(240, 241, 255);
    border-radius: 45px;
}

QPushButton:hover
{
    background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #6a3093, 
        stop:1 #a044ff 
    );
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="IMG.qrc">
      <normaloff>:/IMG/IMG/ExitPin80x80SN.png</normaloff>:/IMG/IMG/ExitPin80x80SN.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="shortcut">
     <string>Ctrl+Z</string>
    </property>
   </widget>
   <widget class="QLabel" name="loginLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	padding: 1px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>100</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton 
{
    background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #6a3093, 
        stop:1 #a044ff 
    );
    color: white;     
    border-radius: 20px;
    padding: 8px 16px;  
    font-weight: bold;   
    border: 1px solid #4a148c; 
}

QPushButton:hover
 {
    background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #7b1fa2,
        stop:1 #aa00ff
    );
    border: 2px solid #7b1fa2;
}

QPushButton:pressed 
{
    background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #4a148c,
        stop:1 #7b1fa2
    );
}</string>
   </property>
   <property name="text">
    <string>Найти</string>
   </property>
  </widget>
  <widget class="QListWidget" name="postList">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>800</width>
     <height>611</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget 
{
	outline: 0;
    border: none;
    background: transparent;
}

QListWidget::item 
{
	border: none;
    background: transparent;
    padding: 0px;
    margin: 0px;
}

QListWidget::item:selected 
{
	background: transparent;
	border: none;
}

QListWidget::item:hover 
{
	background: transparent;
}

QPushButton 
{
    border: none;
    padding: 4px;
}

QScrollBar:vertical 
{
	border: none;
    background: #f5f5f5;
    width: 10px;
    margin: 0px 0px 0px 0px;
}

QScrollBar::handle:vertical 
{
	background: #c1c1c1;
    min-height: 20px;
    border-radius: 5px;
}

QScrollBar::handle:vertical:hover 
{
	background: #a8a8a8;
}

QScrollBar::handle:vertical:pressed
{
	background: #787878;
}

QScrollBar::add-line:vertical 
{
	border: none;
    background: none;
    height: 0px;
    subcontrol-position: bottom;
    subcontrol-origin: margin;
}

QScrollBar::sub-line:vertical 
{
	border: none;
	background: none;
	height: 0px;
	subcontrol-position: top;
	subcontrol-origin: margin;
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical 
{
	background: none;
}

QScrollBar:horizontal 
{
	border: none;
    background: #f5f5f5;
    height: 10px;
    margin: 0px 0px 0px 0px;
}

QScrollBar::handle:horizontal 
{
    background: #c1c1c1;
    min-width: 20px;
	border-radius: 5px;
}

QScrollBar::handle:horizontal:hover 
{
	background: #a8a8a8;
}

QScrollBar::handle:horizontal:pressed 
{
	background: #787878;
}

QScrollBar::add-line:horizontal 
{
    border: none;
    background: none;
    width: 0px;
    subcontrol-position: right;
	subcontrol-origin: margin;
}

QScrollBar::sub-line:horizontal 
{
    border: none;
    background: none;
    width: 0px;
    subcontrol-position: left;
	subcontrol-origin: margin;
}

QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal
{
	background: none;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::NoFrame</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="addPostButton">
   <property name="geometry">
    <rect>
     <x>599</x>
     <y>95</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton 
{
	background-color: white;
    border-radius: 30px;
}

QPushButton:hover
{
    background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #6a3093, 
        stop:1 #a044ff 
    );
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="IMG.qrc">
     <normaloff>:/IMG/IMG/AddPostPin50x50SN.png</normaloff>:/IMG/IMG/AddPostPin50x50SN.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="searchText">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>100</y>
     <width>371</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
	background-color: white;
	color: black;
	border: 1px solid #4a148c; 
	border-radius: 20px;
	padding-left: 20px;
}</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>100</number>
   </property>
   <property name="placeholderText">
    <string>Поиск по названию...</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>70</y>
     <width>1001</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: black;</string>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>174</y>
     <width>1001</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: black;</string>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QFrame" name="commentPostFrame">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>800</width>
     <height>611</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame
{
    background-color: white;
    border: none;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton 
{
	background-color: white;
    border-radius: 45px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="IMG.qrc">
      <normaloff>:/IMG/IMG/BackPin60x60SN.png</normaloff>:/IMG/IMG/BackPin60x60SN.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="commentLineEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>570</y>
      <width>741</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
	padding: 5px;
	border: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Напишите комментарий...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendCommentButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>570</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton 
{
	background-color: white;
    border-radius: 45px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="IMG.qrc">
      <normaloff>:/IMG/IMG/SendCommentMsgPin60x60SN.png</normaloff>:/IMG/IMG/SendCommentMsgPin60x60SN.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>45</width>
      <height>45</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="namePostLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>731</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="countCommentsPostLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>731</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	color: grey;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="commentList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>781</width>
      <height>511</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QScrollBar:vertical 
{
	border: none;
    background: #f5f5f5;
    width: 10px;
    margin: 0px 0px 0px 0px;
}

QScrollBar::handle:vertical 
{
	background: #c1c1c1;
    min-height: 20px;
    border-radius: 5px;
}

QScrollBar::handle:vertical:hover 
{
	background: #a8a8a8;
}

QScrollBar::handle:vertical:pressed
{
	background: #787878;
}

QScrollBar::add-line:vertical 
{
	border: none;
    background: none;
    height: 0px;
    subcontrol-position: bottom;
    subcontrol-origin: margin;
}

QScrollBar::sub-line:vertical 
{
	border: none;
	background: none;
	height: 0px;
	subcontrol-position: top;
	subcontrol-origin: margin;
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical 
{
	background: none;
}

QScrollBar:horizontal 
{
	border: none;
    background: #f5f5f5;
    height: 10px;
    margin: 0px 0px 0px 0px;
}

QScrollBar::handle:horizontal 
{
    background: #c1c1c1;
    min-width: 20px;
	border-radius: 5px;
}

QScrollBar::handle:horizontal:hover 
{
	background: #a8a8a8;
}

QScrollBar::handle:horizontal:pressed 
{
	background: #787878;
}

QScrollBar::add-line:horizontal 
{
    border: none;
    background: none;
    width: 0px;
    subcontrol-position: right;
	subcontrol-origin: margin;
}

QScrollBar::sub-line:horizontal 
{
    border: none;
    background: none;
    width: 0px;
    subcontrol-position: left;
	subcontrol-origin: margin;
}

QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal
{
	background: none;
}</string>
    </property>
   </widget>
   <widget class="QFrame" name="editCommentFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>509</y>
      <width>801</width>
      <height>62</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="editLabel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>5</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
	color: purple;
}</string>
     </property>
     <property name="text">
      <string>Редактирование комментария</string>
     </property>
    </widget>
    <widget class="QLabel" name="editCommentLabel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>671</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelEditComment">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>0</y>
       <width>51</width>
       <height>61</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton 
{
    border-radius: 15px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="IMG.qrc">
       <normaloff>:/IMG/IMG/CancelPin512x512SN.png</normaloff>:/IMG/IMG/CancelPin512x512SN.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="editButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>7</y>
       <width>41</width>
       <height>51</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton 
{
    border-radius: 15px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="IMG.qrc">
       <normaloff>:/IMG/IMG/EditMsg512x512SN.png</normaloff>:/IMG/IMG/EditMsg512x512SN.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
  <zorder>openMenuButton</zorder>
  <zorder>notificationButton</zorder>
  <zorder>updateFeedButton</zorder>
  <zorder>searchButton</zorder>
  <zorder>postList</zorder>
  <zorder>addPostButton</zorder>
  <zorder>searchText</zorder>
  <zorder>line</zorder>
  <zorder>menuTableFrame</zorder>
  <zorder>line_2</zorder>
  <zorder>commentPostFrame</zorder>
 </widget>
 <resources>
  <include location="IMG.qrc"/>
 </resources>
 <connections/>
</ui>
